dotnet-platform-specific/RetrievalOfCpuCores/RetrievalOfCpuCores/CpuInformation.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Runtime.InteropServices;
+
+namespace RetrievalOfCpuCores;
+
+public class CpuInformation
+{
+    /// <summary>
+    /// Retrieves the number of logical processors of the CPU by using Environment.ProcessorCount.
+    /// </summary>
+    /// <returns>number of logical processors</returns>
+    public int GetProcessorsCount()
+    {
+        return Environment.ProcessorCount;
+    }
+
+    /// <summary>
+    /// Retrieves the number of logical processors of the CPU.
+    /// </summary>
+    /// <returns>number of logical processors</returns>
+    public int GetLogicalProcessors()
+    {
+        if (!OperatingSystem.IsWindows())
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        var wmiQuery = "Select * from Win32_ComputerSystem";
```

Comment:
@lennart02 
Can we make this a `const` string and use it in other places in the code. 
This string `"Select * from Win32_ComputerSystem";` is also in the other method.